packages/components/tag/README.md

```diff
@@ -1,43 +1,60 @@
-| `styles.body`          | `object` |    -     | -                                  | -        | Styles spread onto the tag body                                    |
-| `linkTo`               | `string` |    -     | -                                  | -        | Location the tag links to when enabled                             |
-| `isDisabled`           | `bool`   |    -     | -                                  | `false`  | Disables the tag and the option to remove                          |
-| `onRemove`             | `func`   |    -     | -                                  | -        | Called when remove is clicked                                      |
-| `onClick`              | `func`   |    -     | -                                  | -        | Called when tag is clicked (but not when remove button is clicked) |
-| `horizontalConstraint` | `object` |    -     | `xs`, `s`, `m`, `l`, `xl`, `scale` | `scale`  | Horizontal size limit of the input field.                          |
-| `children`             | `node`   |    ✅    | -                                  | -        | Content rendered within the tag                                    |
+```
+yarn add react react-router-dom
+```
+
+```
+npm --save install react react-router-dom
+```
 
-## Where to use
+## Usage
 
-Main Functions and use cases are:
+```jsx
+import React from 'react';
+import Tag from '@commercetools-uikit/tag';
 
-- Adding something to a set _example: Adding products to categories_
+const Example = () => (
+  <Tag
+    type="normal"
+    isDisabled={false}
+    linkTo="/project-key/products/icecream"
+    onRemove={() => {}}
+  >
+    Icecream
+  </Tag>
+);
+
+export default Example;
+```
 
-- Remove from a set _example: Removing products from categories_
+## Properties
 
-- Showing a set _example: List of categories a product is in_
+| Props                  | Type                                                                         | Required | Default         | Description                                                                           |
+| ---------------------- | ---------------------------------------------------------------------------- | :------: | --------------- | ------------------------------------------------------------------------------------- |
+| `type`                 | `enum`<br>Possible values:<br>`'normal' \| 'warning'`                        |          | `'normal'`      | Indicates color scheme of the tag.                                                    |
+| `styles`               | `object`                                                                     |          | `{ body: {}, }` | Styles object spread onto the tag body..                                              |
+| `styles.body`          | `object`                                                                     |          | `{ body: {}, }` |                                                                                       |
```

Comment:
I think we can remove the `styles: { body: {} }` from the default props.

Reply:
Yep, and this should "fix" the Percy diff